packages/design-system/src/components/FilterChip/FilterChip.tsx

```diff
@@ -26,15 +26,15 @@ export interface FilterChipProps {
   /**
    * Function to call when filter chip is dismissed.
    */
-  onDelete: () => void;
+  onDelete: () => any;
   /**
    *  Use alternate thinner close icon in place of standard.
    */
   useAlternateIcon?: boolean;
   /**
    * Sets the size of the chip to larger version.
    */
-  size?: 'big';
+  size?: string;
```

Comment:
Does changing these types cause a breaking change?

Reply:
I don't believe so, it's just making the type looser. There were some typing issues with having this as a string, I could revisit what exactly it was but this was the fix that worked for me to get it going, I could probably resolve the issue without changing this file though, but this is a simple way to do it.

Reply:
The `onDelete` change is harmless, but this one with the `size` causes devs to lose the hint in their code editor as to what one string value is allowed here.